tutor4points/core/templates/home.html

```diff
@@ -70,13 +100,88 @@ <h2>Suggested Sessions</h2>
                         {% endfor %}
                 </div>
             </div>
+            <div class="dashboard-container">
+                <div class="list-container">
+                    <h2>Sent Requests</h2>
```

Comment:
I think it'll be helpful if clicking on the sent requests leads to the sent request page. Also, it'll be helpful to show the status of the request, such as "Accepted, Declined, Pending"

Reply:
True. I can put in a dummy link until the sent requests page is created. I'll work on showing the request status for Sent Requests.